game/scripts/vscripts/units/twin.lua

```diff
@@ -0,0 +1,29 @@
+LinkLuaModifier( "modifier_boss_phase_controller", "modifiers/modifier_boss_phase_controller", LUA_MODIFIER_MOTION_NONE )
+
+function Spawn (entityKeyValues)
+  thisEntity:FindAbilityByName("boss_twin_twin_empathy")
+  thisEntity:FindAbilityByName("boss_twin_spawn_twin")
+
+  thisEntity:SetContextThink( "TwinThink", partial(TwinThink, thisEntity) , 1)
+  print("Starting AI for " .. thisEntity:GetUnitName() .. " " .. thisEntity:GetEntityIndex())
+
+  ABILITY_spawn_twin = thisEntity:FindAbilityByName("boss_twin_spawn_twin")
```

Comment:
Since we don't need to use an ability to spawn the twin, we can just remove this.